src/cljs/org/broadinstitute/firecloud_ui/page/profile.cljs

```diff
@@ -12,6 +12,13 @@
    ))
 
 
+(defn get-nih-link-href []
+  (str (get @config/config "shibbolethUrlRoot")
+       "/link-nih-account?redirect-url="
+       (js/encodeURIComponent
+        (str (-> js/window .-location .-href) "/nih-username-token={token}"))))
+
+
```

Comment:
Something isn't quite right here ... I re-linked my account and ended up at:
<img width="1429" alt="firecloud___broad_institute" src="https://cloud.githubusercontent.com/assets/6041577/12207557/566791b2-b615-11e5-9f2e-095bb42b1e00.png">